tests/js/session-tests.js

```diff
@@ -1182,11 +1182,18 @@ module.exports = {
     await realm1.syncSession.uploadAllLocalChanges();
     await realm1.syncSession.downloadAllServerChanges();
 
-    // changes are synced -- we should be able to copy the realm
-    realm1.writeCopyTo(realm1Path + "copy1.realm");
+    const outputConfig1 = {
+      sync: {
+        user: user1,
+        partitionValue: partition,
+        _sessionStopPolicy: "immediately", // Make it safe to delete files after realm.close()
+      },
+      schema: [schemas.PersonForSync, schemas.DogForSync],
+      path: `${realm1Path}copy1.realm`,
+    };
 
-    // log out the user that created the realm
-    await user1.logOut();
```

Comment:
Is this not an intended part of the test? Why did this logout get removed?

Reply:
It breaks the test.  We can't copy a synced realm without a valid user session

Reply:
Any idea (if /) why that worked before?

Reply:
Ah!  I can convert it to a local realm after user logout.  I'll test this